src/node.js

```diff
@@ -118,51 +117,106 @@ module.exports = class Node {
     }
   }
 
-  startDaemon (done) {
-    parseConfig(this.path, (err, conf) => {
+  startDaemon (flags, done) {
+    if (typeof flags === 'function' && typeof done === 'undefined') {
+      done = flags
+      flags = []
+    }
+
+    const node = this
```

Comment:
Seems that having access to the `this` of this func is not necessary. Rather use `{}` to initialize it instead?